packages/css-module/package.json

```
@@ -13,6 +14,9 @@
     "loader",
     "node.js"
   ],
+  "engines": {
+    "node": ">=22"
```

Why? I think same as recs for module.register

```suggestion
    "node": ">=22 || ^20.6.0 || ^18.19.0"
```